src/flag/model.rs

```diff
@@ -73,6 +71,94 @@ impl ToString for Flags {
     }
 }
 
+impl ToString for Flags {
+    fn to_string(&self) -> String {
+
```

Comment:
Why adding a new line here? It does not really fit the global style of the code base.